docs/driver_installation_zadig.md

```diff
@@ -8,10 +8,10 @@ We recommend the use of the [Zadig](https://zadig.akeo.ie/) utility. If you have
 
 ## Installation
 
-Place your keyboard into the bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch usually located on the underside of the board. If your keyboard has neither, try holding Escape, or Space+`B`, as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details).
-Some keyboards may have specific instructions for entering the bootloader, for example the [Bootmagic Lite](feature_bootmagic.md#bootmagic-lite) key (Escape) might be on a different key, such as Left Control. Refer to the board's README if you are unsure.
+Put your keyboard into bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch that's usually located on the underside of the board. If your keyboard has neither, try holding Esc or Space+B as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details). Some boards use [Command](feature_command.md) instead of Bootmagic; in this case, you can enter bootloader mode by hitting LShift+RShift+B or LShift+RShift+Esc at any point while the keyboard is plugged in.
```

Comment:
```suggestion
Put your keyboard into bootloader mode, either by hitting the `RESET` keycode (which may be on a different layer), or by pressing the reset switch that's usually located on the underside of the board. If your keyboard has neither, try holding Escape or Space+`B` as you plug it in (see the [Bootmagic](feature_bootmagic.md) docs for more details). Some boards use [Command](feature_command.md) instead of Bootmagic; in this case, you can enter bootloader mode by hitting Left Shift+Right Shift+`B` or Left Shift+Right Shift+Escape at any point while the keyboard is plugged in.
```

I'd like to see keys referred to using their full names as defined in keycodes.md (within reason), with printable characters enclosed in code tags.

Reply:
Sounds good. I had glossed over the fact that this is an established convention in the docs. Will fix right away.

Reply:
It's not, really, I just want it to be 😁 IMO it just makes the docs feel more polished if they reference keys and keycodes with a consistent style.

Reply:
Agreed, consistency is one of my favorite things ever. Though I don't particularly like putting the printable characters into code quotes. I think the best solution for this would be to have custom markup (akin to the `?>`, `!>` markup we already use) for computer keys, similar to what [Wikipedia has](https://en.wikipedia.org/wiki/Alt_key#Alt_key_combinations). It shouldn't be too big of an issue to implement this docs-wide.

Reply:
Either way, I've updated my changes to match the existing convention we have going on, so this should be good to go!

Reply:
Yeah, the `<kbd>` tag would be better (and could be used for non-printable keys too), but there is no nice way to do it in Markdown other than using the tag directly.